agent/src/test/java/com/criteo/hadoop/garmadon/agent/tracers/hadoop/hdfs/FileSystemTracerTest.java

```diff
@@ -180,47 +187,99 @@ private void checkEvent(String action, Path path, DataAccessEventProtos.FsEvent.
         assertEquals(status, eventTmp.getStatus());
     }
 
+    private void createSrcPathHdfs(Path file) throws NoSuchMethodException, InvocationTargetException, IllegalAccessException, ClassNotFoundException {
+        Method create = clazzAFS.getMethod("createInternal", Path.class,
+            EnumSet.class,
+            FsPermission.class,
+            int.class,
+            short.class,
+            long.class,
+            Progressable.class,
+            Options.ChecksumOpt.class,
+            boolean.class);
+
+        //EnumSet<CreateFlag> createFlag
+        Object os = create.invoke(hdfs, file, EnumSet.of(CreateFlag.CREATE), FsPermission.getDefault(), 1024, (short) 1, 1048576, null,
+            Options.ChecksumOpt.createDisabled(), false);
+
+        Class<?> clazzHdfsDataOutputStream = classLoader.loadClass(HdfsDataOutputStream.class.getName());
+        Method write = clazzHdfsDataOutputStream.getMethod("write", byte[].class);
+        Method close = clazzHdfsDataOutputStream.getMethod("close", null);
+        write.invoke(os, "This is a test".getBytes());
+        close.invoke(os);
+    }
+
+    private void createSrcPatDistributedFilesystem(Path file) throws NoSuchMethodException, InvocationTargetException, IllegalAccessException, ClassNotFoundException, IOException {
```

Comment:
typo missing h to `createSrcPatDistributedFilesystem`

Reply:
in fact will do